foundations/javascript_basics/revisiting_rock_paper_scissors.md

```diff
@@ -18,7 +18,7 @@ You can make new branches by using the command `git branch <branch_name>`. You c
 
 You can see all of your current branches using `git branch` with no other arguments. The branch that you’re currently on will be indicated with an asterisk. If you want to change back to `main` from any other branch, you can do so just like changing to any other branch using `git checkout main`.
 
-Once you’re done working on your feature branch and you’re ready to bring the commits that you’ve made on it to your main branch,  you need to perform what is known as a `merge`.
+Once you’re done working on your feature branch and ready to bring the commits that you’ve made on it to your main branch,  you need to perform what is known as a `merge`.
```

Comment:
This sentence reads weird to me, in a few places. Here is a suggestion, feel free to update it. (also removed two spaces after a comma)
```suggestion
Once you are done working on your feature branch and are ready to bring the commits that you’ve made on it to your main branch, you will need to perform what is known as a `merge`.
```